### PR TITLE
Only front

### PR DESCRIPTION
**Pull request by AleksanderGorazda**

### CODE REVIEW

File: API-ERP/API-ERP.csproj.user [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <ActiveDebugProfile>Docker</ActiveDebugProfile>
+    <ActiveDebugProfile>IIS Express</ActiveDebugProfile>
```
Comment (OskarKozaczka) [inline API-ERP/API-ERP.csproj.user]: Do we need this?

**Review by OskarKozaczka — CHANGES_REQUESTED** [review]

File: API-ERP/API-ERP.csproj.user (hunk 1 above)
Comment (AleksanderGorazda) [reply API-ERP/API-ERP.csproj.user]: yes this is neccesary please approve

**Review by OskarKozaczka — APPROVED** [review]
kozak

File: API-ERP/API-ERP.csproj.user (hunk 1 above)
Comment (OskarKozaczka) [reply API-ERP/API-ERP.csproj.user]: git

**Pull request merged**